src/theme/overrides/MuiButton.js

```diff
@@ -1,49 +1,62 @@
 import palette from '../palette'
 
-const MuiButton = {
-  contained: {
-    boxShadow: 'none',
-    '&:disabled': {
-      backgroundColor: palette.primary.main,
-      color: palette.common.white,
-      opacity: 0.5,
-    },
-    '&:hover': {
+const MuiButton = (color) => {
+  const customColor = (color || 'primary')
+
+  return {
+    contained: {
       boxShadow: 'none',
+      '&:disabled': {
+        backgroundColor: palette[customColor].main,
+        color: palette.common.white,
+        opacity: 0.5,
+      },
+      '&:hover': {
+        boxShadow: 'none',
+      },
     },
-  },
-  containedPrimary: {
-    '&:hover': {
-      backgroundColor: palette.primary[800],
+    containedPrimary: {
+      backgroundColor: palette[customColor].main,
+      '&:disabled': {
+        backgroundColor: palette[customColor].main,
+        color: palette.common.white,
```

Comment:
@tlowande good catch!

Reply:
just added the text color based on the background color for button from chip.js. Probable should make it as a Shareable function so it can be used globally. Its is pretty useful if more custom color needs to be added in the future for other components. What do you guys think???

Reply:
That sounds good to me. Go ahead! 👍 What identifies the color contrast is the `determineColor` above the code Tamires mentioned.
```jsx
const determineColor = backgroundColor => {
  const ratioToWhite = getContrastRatio(backgroundColor, '#FFF')
  const ratioToBlack = getContrastRatio(backgroundColor, '#000')
  return ratioToWhite >= ratioToBlack ? '#FFF' : '#000'
}
```

Reply:
@DoParkEQ & @tlowande I have added a new folder "util" for the basics shareable functions and resolved your feedbacks. Let me know if there is anything I could improve.